sdk-ts/package.json

```diff
@@ -35,7 +35,7 @@
     "@project-serum/serum": "^0.13.60",
     "@pythnetwork/client": "^2.5.1",
     "@solana/spl-token": "^0.1.8",
-    "@solana/web3.js": "^1.31.0",
+    "@solana/web3.js": "1.31.0",
```

Comment:
I've tried both yarn and npm. It installs v1.33.0 and breaks PriceInfo build.

yarn.lock
```
"@solana/web3.js@^1.17.0", "@solana/web3.js@^1.21.0", "@solana/web3.js@^1.30.2", "@solana/web3.js@^1.31.0":
  version "1.33.0"
  resolved "https://registry.yarnpkg.com/@solana/web3.js/-/web3.js-1.33.0.tgz#3b8c0b4cd259ffe8764937f1001f79958c5d9533"
  integrity sha512-R53wvQQsUKCCJ9UmOfDlxCwC94jzjmpjr6wT/Xf1uFavZblSLAtzLaF4vbGIS465lk3mW6oJMtASd8cqSnr8Mw==
  dependencies:
    "@babel/runtime" "^7.12.5"
    "@ethersproject/sha2" "^5.5.0"
    "@solana/buffer-layout" "^3.0.0"
    bn.js "^5.0.0"
    borsh "^0.4.0"
    bs58 "^4.0.1"
    buffer "6.0.1"
    cross-fetch "^3.1.4"
    jayson "^3.4.4"
    js-sha3 "^0.8.0"
    rpc-websockets "^7.4.2"
    secp256k1 "^4.0.2"
    superstruct "^0.14.2"
    tweetnacl "^1.0.0"
```

Build error:
```
src/utils/PriceInfo.ts:158:29 - error TS2339: Property 'length' does not exist on type 'Buffer | ParsedAccountData'.
  Property 'length' does not exist on type 'ParsedAccountData'.

158         invariant(info.data.length === AccountLayout.span, 'Invalid token account info data length');
                                ~~~~~~

src/utils/PriceInfo.ts:163:29 - error TS2339: Property 'length' does not exist on type 'Buffer | ParsedAccountData'.
  Property 'length' does not exist on type 'ParsedAccountData'.

163         invariant(info.data.length === MintLayout.span, 'Invalid mint account info data length');
                                ~~~~~~

src/utils/PriceInfo.ts:171:31 - error TS2339: Property 'length' does not exist on type 'Buffer | ParsedAccountData'.
  Property 'length' does not exist on type 'ParsedAccountData'.

171           invariant(info.data.length === LAYOUT.span, 'Invalid raydium open orders account info data length');
                                  ~~~~~~

src/utils/PriceInfo.ts:177:31 - error TS2339: Property 'length' does not exist on type 'Buffer | ParsedAccountData'.
  Property 'length' does not exist on type 'ParsedAccountData'.

177           invariant(info.data.length === AMM_INFO_LAYOUT_V4.span, 'invalid raydium amm ID account data length');
                                  ~~~~~~

src/utils/PriceInfo.ts:288:29 - error TS2339: Property 'length' does not exist on type 'Buffer | ParsedAccountData'.
  Property 'length' does not exist on type 'ParsedAccountData'.

288         invariant(info.data.length === AccountLayout.span, 'Invalid token account info data length');
                                ~~~~~~

src/utils/PriceInfo.ts:293:29 - error TS2339: Property 'length' does not exist on type 'Buffer | ParsedAccountData'.
  Property 'length' does not exist on type 'ParsedAccountData'.

293         invariant(info.data.length === MintLayout.span, 'Invalid mint account info data length');
                                ~~~~~~

src/utils/PriceInfo.ts:301:31 - error TS2339: Property 'length' does not exist on type 'Buffer | ParsedAccountData'.
  Property 'length' does not exist on type 'ParsedAccountData'.

301           invariant(info.data.length === LAYOUT.span, 'Invalid raydium open orders account info data length');
                                  ~~~~~~

src/utils/PriceInfo.ts:307:31 - error TS2339: Property 'length' does not exist on type 'Buffer | ParsedAccountData'.
  Property 'length' does not exist on type 'ParsedAccountData'.

307           invariant(info.data.length === AMM_INFO_LAYOUT_V4.span, 'invalid raydium amm ID account data length');
                                  ~~~~~~


Found 8 errors.
```